### ISSUE TITLE
Does not receives incoming payments

### ISSUE THREAD

**Issue by MrSamUA** [amjuarez/bytecoin-gui#8]
Bytecoin Wallet does not show incoming payments that are already confirmed in blockchain explorer.

ByteCoin Wallet - v1.1.9.3
Remote node daemon - node.bytecoin.ninja:8081
Wallet synchronized.

**Comment by MikeFoden** [comment]
Does yours show up as being Disconnected in the status bar at the bottom?

![bytecoin](https://user-images.githubusercontent.com/177184/34655691-fc443130-f461-11e7-97cb-5bf8aed316f1.png)

**Comment by MrSamUA** [comment]
No. It says "Wallet synchronized. Top block height: xxxxxxx. ...... 9 peers" (xxxxx is much higher than blocks with my payments).

**Comment by siliconlumen** [comment]
My wallet is also synchronised but shows Disconnected at the bottom right. Do you have a solution for this ? Firewall on desktop has been enabled but doesn't solve it.

**Comment by MrSamUA** [comment]
Nope.. I think that's client's bug (I have the same problem with CLI). Devs are preparing "Tech Update".. I hope it'll fix that.
( https://bytecoin.org/blog/a-bit-more-transparency-on-the-current-development-situation )

**Comment by siliconlumen** [comment]
In the Wallet.log file I see this:  **[BlockchainSynchronizer] Failed to query blocks: NodeErrorCategory:5, Internal node error**, the simplewallet works fine but the QtWallet Fork I made connects, sees 2-3 peers, shows the correct BC_height, but no coins are being transferred to the qtwallet. I even rebuild the DB and started over but same problem.
